.pipelines/diabetes_regression-ci-build-train.yml

```diff
@@ -1,3 +1,4 @@
+# CI pipeline that builds, trains, tests, and deploys the diabetes model.
```

Comment:
>diabetes [](start = 58, length = 8)

diabetes_regression (bootstrap script should update this to the project name)

Reply:
That file is already covered in bootstrap, sounds good.